src/test/scala/edu/ie3/simona/agent/grid/PowerFlowSupportSpec.scala

```diff
@@ -31,12 +35,16 @@ import javax.measure.quantity.Angle
   * order to comprehend the expected test results.
   */
 class PowerFlowSupportSpec
-    extends UnitSpec
+    extends ScalaTestWithActorTestKit
+    with UnitSpec
     with BasicGridWithSwitches
     with PowerFlowSupport
     with GridResultsSupport {
 
-  override val log: LoggingAdapter = NoLogging
+  implicit val log: Logger =
+    LoggerFactory.getLogger(PowerFlowSupportSpec.super.getClass)
```

Comment:
Just a curious question: Why'd you get the logger for the superclass here?

Reply:
Because ``PowerFlowSupportSpec.getClass`` does not work.

Reply:
`PowerFlowSupportSpec.this.getClass` or `classOf[PowerFlowSupportSpec]` works